sphinx-docs/SCREENSHOT_USAGE.MD

```diff
@@ -0,0 +1,36 @@
+Check out the ss_examples.rst file for some examples. The idea is to have a directive for specifying screenshots of the ka-lite web app that can be automagically updated in the case of UI changes or for building docs in different languages.
```

Comment:
Meaning it doesn't really matter where the sphinx project is and it could be moved elsewhere. We could rst-ify the pages in `docs` and add them to the sphinx project.